brozzler/browser.py

```diff
@@ -457,8 +457,11 @@ def _visit_page(self, websock):
         self.send_to_chrome(method="Debugger.enable")
         self.send_to_chrome(method="Runtime.enable")
 
-        if self.extra_headers:
-            self.send_to_chrome(method="Network.setExtraHTTPHeaders", params={"headers":self.extra_headers})
+        headers = self.extra_headers or {}
+        headers['Accept-Encoding'] = 'gzip, deflate'
```

Comment:
I'm thinking let's change this to 

        headers['Accept-Encoding'] = 'identity'

Because I'm not sure "deflate" is supported by the various wayback implementations (even if it really means gzip[1](https://stackoverflow.com/questions/9170338/why-are-major-web-sites-using-gzip/9186091#9186091))
And of course warcs are usually gzipped anyway, so gzipping inside of them doesn't save much space, and makes it more of a pain to inspect the contents.